jotty/docker-compose.yml

```
@@ -0,0 +1,15 @@
+version: "3.7"
+services:
+  app_proxy:
+    environment:
+      APP_HOST: jotty_web_1
+      APP_PORT: "3000"
+  web:
```

Add newline for better readability.
```suggestion
      APP_PORT: "3000"

  web:
```